.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .DS_Store
 build-boilerplate
 *.tgz
+.vscode
```

Comment:
Do we want launch templates for vscode in the project? This is helpful for debugging tests. But they are only a quick Google search away as well.

Reply:
could be good! that's a good backlog item for the eventual monorepo